packages/components/src/FormField/FormFieldWrapper.tsx

```diff
@@ -125,7 +134,32 @@ export function FormFieldWrapper({
           )}
 
           {prefix?.label && <AffixLabel {...prefix} labelRef={prefixRef} />}
-          <div className={styles.childrenWrapper}>{children}</div>
+
+          <div className={styles.childrenWrapper} tabIndex={-1}>
+            {children}
+            <AnimatePresence>
+              {isToolbarVisible && (
+                <motion.div
+                  key="toolbar"
+                  initial={animationInitial}
+                  animate={{
+                    opacity: 1,
+                    height: "auto",
+                  }}
+                  exit={{ opacity: 0, height: 0 }}
+                  transition={{
+                    duration: tokens["timing-base"] / 1000,
+                    ease: "easeInOut",
+                  }}
+                  tabIndex={-1}
```

Comment:
weird that you had to tabindex -1 it when divs aren't tabbable be default 🤔

Reply:
agreed. I'm not 100% sold on this being the best fix, and it barely even makes sense - but it does fix the problem

without this, clicks on these elements combined with the focus event listener would unset `focused` because the clicks would seemingly go thru the elements and end up on the body somehow